openwisp_radius/migrations/0002_initial_openwisp_radius.py

```diff
@@ -28,12 +29,14 @@ class Migration(migrations.Migration):
     """
 
     dependencies = [
-        ('openwisp_users', '0003_default_organization'),
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('openwisp_radius', '0001_initial_freeradius'),
     ]
 
     operations = [
+        migrations.RunPython(
+            set_default_organization_uuid, reverse_code=migrations.RunPython.noop
+        ),
```

Comment:
@codesankalp: this can cause issues on PostgreSQL, please read:
https://docs.djangoproject.com/en/3.1/ref/migration-operations/#runpython

> On databases that do support DDL transactions (SQLite and PostgreSQL), RunPython operations do not have any transactions automatically added besides the transactions created for each migration. Thus, on PostgreSQL, for example, you should avoid combining schema changes and RunPython operations in the same migration or you may hit errors like OperationalError: cannot ALTER TABLE "mytable" because it has pending trigger events.

This has already happened a few times to me and other contributors (eg: @atb00ker, @pandafy).

I think we have two options:

1. find a way to run the migrations of OpenWISP RADIUS only once all the migrations of OpenWISP Users have been run, this can be achieved by referring to `__latest__`, but we won't be able to use `migrations.swappable_dependency` because it points to `__first__` (for good reasons), see the [django source code](https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/db/migrations/migration.py#L216-L218) for reference. I think we should implement an alternative to the function just linked which does the same but points to `__latest__`; 
2. run the data migration in https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/migrations/0003_default_radius_groups.py, which doesn't do schema changes.

I think we should look for a generic solution that we can reuse in other modules, since the problem will appear there too. That's why I think option 1 may be the best for our use case since once we make it work we just have to use it in the same way in other modules.
Referring to `__latest__` can have drawbacks, however I believe that in our case we will be fine because we only add foreign keys to models like `User` and `Organization` and do not refer to anything else which may change abrutly. We can consider these two models as stable.